core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -190,15 +190,16 @@ export class WorkflowActionService {
     // check that the operator doesn't exist
     this.texeraGraph.assertOperatorNotExists(operator.operatorID);
     // check that the operator type exists
-    if (!this.operatorMetadataService.operatorTypeExists(operator.operatorType)) {
-      throw new Error(`operator type ${operator.operatorType} is invalid`);
-    }
-
-    this.texeraGraph.bundleActions(() => {
-      // add operator to texera graph
-      this.texeraGraph.addOperator(operator);
-      this.texeraGraph.sharedModel.elementPositionMap?.set(operator.operatorID, point);
-    });
+    this.operatorMetadataService
+      .operatorTypeExists(operator.operatorType)
+      .pipe(filter(exist => exist))
```

Comment:
I'm strongly against this change. This might silently cover up potential issues.

For example for some reason the metadata isn't loaded. After this change there is no error in loading a workflow, but none of the operators are actually added. Then this wrong empty workflow can saved and override the user's original workflow content. This is very bad

Reply:
Understood. I can add logic to throw errors.